Model/Config.php

```diff
@@ -118,6 +138,20 @@ public function getHub()
             \Sentry\init([
                 'dsn' => $this->getConnection(),
                 'environment' => $this->getEnvironment() ?? null,
+                'before_send' => function (\Sentry\Event $event, ?\Sentry\EventHint $hint): ?\Sentry\Event {
```

Comment:
unused $hint
Can we remove it?
https://docs.sentry.io/platforms/php/configuration/filtering/

Reply:
Sure, removed.